task_12/dist/app.bundle.js

```diff
@@ -1630,6 +1607,112 @@ class Time {
 
 module.exports = __webpack_require__.p + "images/pizza-PomoLivo.png";
 
+/***/ }),
+
+/***/ "./src/store/login.service.js":
+/***/ (function(module, __webpack_exports__, __webpack_require__) {
+
+"use strict";
+class AuthService {
+    constructor(){
+        this._token = localStorage.getItem('token');
+        this._claims = JSON.parse(localStorage.getItem('claims'));
+    }
+
+    get token(){
+        return this._token;
+    }
+    set token(token){
+        this._token = token;
+        localStorage.setItem('token', token);
+    }
+
+    get claims(){
+        return this._claims;
+    }
+    
+    set claims(claims){
+        this._claims = claims;
+        localStorage.setItem('claims', JSON.stringify(claims));
+    }
+
+    clearStorage(){
+        this._token = null;
+        this._claims = null;
+        localStorage.removeItem('token');
+        localStorage.removeItem('claims');
+    }
+
+    isAuthorized(){
+        if( !this.tokenIsNotExpired() ){
+            this.clearStorage();
+            return false;
+        }
+        return true;
+    }
+
+    tokenIsNotExpired(){
```

Comment:
for this function to return `true`, you need two conditions to be met: `this.token` must exist and `this.claims.exp * 1000` must be greater than Date.now(). All other combinations of values should return `false`. So what you can do is like so:
`return this.token && this.claims.exp * 1000 > Date.now();`